test/AuthorizationHeaderTest.php

```diff
@@ -104,34 +112,54 @@ public function testCaseInsensitiveRequestMethod()
 
     /**
      * Ensures an exception is thrown if a request does not have an Authorization header.
-     *
-     * @expectedException \Acquia\Hmac\Exception\MalformedRequestException
      */
     public function testCreateFromRequestNoAuthorizationHeader()
     {
+        $this->setExpectedException(
```

Comment:
Is switching to `setExpectedException()` from the `@expectedException` annotation necessary for the test to pass? If it's just to test the message, you can use the `@expectedExceptionMessage` annotation.